guides/release/tutorial/ember-data.md

```diff
@@ -50,6 +51,7 @@ export default DS.Model.extend({
 });
 ```
 
+If you are curious about that `{ Model }` syntax above, it is called ES6 [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) with `const` declaration, and it is a feature of JavaScript.
```

Comment:
@DPK101 great find!
You can actually use destructuring with any kind of variable declaration:
```js
const pet = {
  details: {
    name: 'fred'
  }
};

let { details: { name } } = pet;
console.log(name);
```
Perhaps we can remove the  `with const declaration` qualification. As an aside, we could consider adding destructuring as an entry in the [glossary](https://github.com/ember-learn/guides-source/blob/master/guides/release/glossary/web-development.md).

Reply:
Nice catch @efx 
I have done the correction.